src/Backend/Haskell.idr

```diff
@@ -122,6 +122,11 @@ Backend Haskell where
   generateCode        = renderDef
   freshEnv            = freshEnvLC
 
+NewBackend Haskell HsType where
```

Comment:
Is the absence if `NewBackend` instance for ReasonML intentional?

Reply:
Kind of? I'm not even sure why I added the instance for `Haskell` tbh, I think it was mostly to check that `NewBackend` actually is compatible with the old backends. I mean, neither `Backend` nor `NewBackend` will be kept as-is, so I didn't see the point on spending too much time on them. *But* now that you ask, I realize that it's good hygiene that it's good to keep `master` "complete" at all times, so I'll add a `ReasonML` instance.